adafruit_hid/keyboard.py

```diff
@@ -22,6 +22,11 @@
 class Keyboard:
     """Send HID keyboard reports."""
 
+    LED_NUM_LOCK = 0x01
+    LED_CAPS_LOCK = 0x02
+    LED_SCROLL_LOCK = 0x04
+    LED_COMPOSE = 0x08
```

Comment:
Please add documentation for each of these as a docstring after the assignment.

Reply:
I have added docstrings after each assignment.